dist/pathfora.js

```diff
@@ -2910,7 +2910,19 @@
 
       this.getData(recommendUrl, function (json) {
         var resp = JSON.parse(json);
+
         if (resp.data && resp.data.length > 0) {
+          // append a protocol for urls that are absolute
+          for (var i = 0; i < resp.data.length; i++) {
+            var url = resp.data[i].url;
+            if (url) {
+              var split = url.split('/')[0].split('.');
+              if (split.length > 1) {
+                resp.data[i].url = 'http://' + url;
```

Comment:
Yup, as far as I can tell it's always been like that. Since we don't have the protocol in the API, we can't assume that it is secure or the same as the current page. I think we found that http was "safest" in that most websites that use https will set up a redirect. Would be great if we could have the protocol as part of the URL from the API. I believe @drewlanenga & @DingoEatingFuzz have had conversations around that already.